hist/histv7/inc/ROOT/RHistImpl.hxx

```diff
@@ -386,7 +621,11 @@ public:
 
    /// Retrieve the fill function for this histogram implementation, to prevent
    /// the virtual function call for high-frequency fills.
-   FillFunc_t GetFillFunc() const final { return (FillFunc_t)&RHistImpl::Fill; }
+   FillFunc_t GetFillFunc() const final { 
+      auto testFill = (FillFunc_t)&RHistImpl::Fill;
+      testFill = nullptr;
```

Comment:
This dead code makes my compiler very unhappy when I try to compile this branch locally. If you don't need it anymore, please consider removing it.